app/views/posts/show.html.erb

```diff
@@ -5,7 +5,22 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
   </head>
   <body>
-    <h1><%= @post.title %></h1> <!-- Display the post title here -->
-    <p>Placeholder: This is the details page for a specific post.</p>
+  <div class="container">
+    <div class="user-header">
+        <h2><%= @user.name %></h2>
+        <h3><%= @targated_post.title %></h3>
+        <p>Number of Comments: <%= @targated_post.comments_counter %></p>
+    </div>
+    
+    <div class="sectioncomment">
+        <h3>Comments</h3>
+        <ul>
+            <% @comments.each do |comment| %>
+                <li><%= comment.text %></li>
+            <% end %>
+        </ul>
+		<a href="posts">Back to Posts</a>
+    </div>
+</div>
```

Comment:
- [ ] Kindly fix the route to the post-show page, as I am unable to access the page, I get the following error
![image](https://github.com/Gardimy/BlogApp/assets/81932478/55c210d0-147b-4426-b10a-ad6c050c46b4)

Kindly also ensure when you do that you fix your design to match the given layout below
![image](https://github.com/Gardimy/BlogApp/assets/81932478/a2d9e01e-0770-4203-bb2d-42a8728909f4)